ta/pkcs11/src/processing.c

```diff
@@ -79,6 +79,15 @@ void release_active_processing(struct pkcs11_session *session)
 	case PKCS11_CKM_AES_CTR:
 		tee_release_ctr_operation(session->processing);
 		break;
+	case PKCS11_CKM_MD5_HMAC:
+	case PKCS11_CKM_SHA_1_HMAC:
+	case PKCS11_CKM_SHA224_HMAC:
+	case PKCS11_CKM_SHA256_HMAC:
+	case PKCS11_CKM_SHA384_HMAC:
+	case PKCS11_CKM_SHA512_HMAC:
+		if (session->processing->extra_ctx)
```

Comment:
`session->processing->extra_ctx` is a generic context pointer so I think freeing it could be done in the generic path, like right before line 100 below.

Reply:
You are right. I have moved it.